From ed5fcc718075021fe670daee6dd426a5e12d6842 Mon Sep 17 00:00:00 2001
From: Alyssa Ross <hi@alyssa.is>
Date: Thu, 19 Dec 2024 16:51:51 +0100
Subject: [PATCH] Fix name of Lerc package

Lerc calls its CMake package "Lerc", not "LERC".  This meant that,
even if building libtiff with -DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON,
Lerc's CMake files wouldn't be used as intended.

Link: https://gitlab.com/libtiff/libtiff/-/merge_requests/686
---
 cmake/{FindLERC.cmake => FindLerc.cmake} | 0
 cmake/LERCCodec.cmake                    | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename cmake/{FindLERC.cmake => FindLerc.cmake} (100%)

diff --git a/cmake/FindLERC.cmake b/cmake/FindLerc.cmake
similarity index 100%
rename from cmake/FindLERC.cmake
rename to cmake/FindLerc.cmake
diff --git a/cmake/LERCCodec.cmake b/cmake/LERCCodec.cmake
index 54504ca1..237f8d7e 100644
--- a/cmake/LERCCodec.cmake
+++ b/cmake/LERCCodec.cmake
@@ -25,7 +25,7 @@
 
 # libLerc
 set(LERC_SUPPORT FALSE)
-find_package(LERC)
+find_package(Lerc)
 option(lerc "use libLerc (required for LERC compression)" ${LERC_FOUND})
 if (lerc AND LERC_FOUND AND ZIP_SUPPORT)
     set(LERC_SUPPORT TRUE)
-- 
2.47.0

